### PR TITLE
Remove template editor from Slack

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does
Screenshot 2023-04-28 at 5 36 10 PM

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

**Review by Konstantinov-Innokentii — APPROVED** [review]

File: engine/apps/slack/scenarios/alertgroup_appearance.py [hunk 1]
```diff
@@ -45,148 +39,17 @@ def process_scenario(self, slack_user_identity, slack_team_identity, payload):
...
-            else:
-                default_values = getattr(
-                    AlertReceiveChannel, f"INTEGRATION_TO_DEFAULT_{common_template.upper()}_TEMPLATE", None
-                )
-                if default_values is not None:
-                    default_value = default_values.get(integration)
-                    if default_value:
-                        block["element"]["initial_value"] = default_value
-            blocks.append(block)
+            {"type": "section", "text": {"type": "mrkdwn", "text": "Once changed Refresh the alert group"}},
```
Comment (Konstantinov-Innokentii) [inline engine/apps/slack/scenarios/alertgroup_appearance.py:51]: Do we really need refresh alert group? We don't have this for msteams and telegram

Comment (iskhakov) [reply engine/apps/slack/scenarios/alertgroup_appearance.py:51]: We don't want to silently remove the button, so keeping this for transition period and then decide

File: engine/apps/slack/scenarios/alertgroup_appearance.py [hunk 2]
```diff
@@ -45,148 +39,17 @@ def process_scenario(self, slack_user_identity, slack_team_identity, payload):
...
-                    },
-                    "action_id": UpdateAppearanceStep.routing_uid(),
-                    "multiline": True,
+        alert_receive_channel = alert_group.channel
+        blocks = [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f":point_right: Click <{alert_receive_channel.web_link}|here> to open Integrations settings, edit Slack templates and return here",
```
Comment (vadimkerr) [inline engine/apps/slack/scenarios/alertgroup_appearance.py:48]: nit: Maybe use a button with a URL as shown in [Slack docs](https://api.slack.com/reference/block-kit/block-elements#button) to redirect to the web UI? A button with URL seems to be more appropriate for this use case

**Review by vadimkerr — APPROVED** [review]

File: engine/apps/slack/scenarios/alertgroup_appearance.py (hunk 2 above)
Comment (iskhakov) [reply engine/apps/slack/scenarios/alertgroup_appearance.py:48]: In the next version we’ll have 3 urls to 3 templates

**Pull request merged**